### ISSUE TITLE
tacc-globalfs still points to gateway node

### ISSUE THREAD

**Issue by zyndagj** [cyverse/cyverse-sdk#18]
https://github.com/cyverse/cyverse-sdk/blob/master/src/templates/tacc-stampede-storage.jsonx

still points to `gateway.stampede.tacc.utexas.edu` which yields authentication errors

```
staff.stampede2(1116)$ jobs-submit -F job.json
Unable to locate application assets on remote system. Remote server responded with: Failed to authenticate to gateway.stampede.tacc.utexas.edu
```

**Comment by mwvaughn** [comment]
Good catch. It is not supposed to be erroring though. I’ll fix it this afternoon till we can diagnose

On Nov 15, 2017, at 11:35 AM, Greg Zynda <notifications@github.com<mailto:notifications@github.com>> wrote:


https://github.com/cyverse/cyverse-sdk/blob/master/src/templates/tacc-stampede-storage.jsonx

still points to gateway.stampede.tacc.utexas.edu<http://gateway.stampede.tacc.utexas.edu> which yields authentication erros

staff.stampede2(1116)$ jobs-submit -F job.json
Unable to locate application assets on remote system. Remote server responded with: Failed to authenticate to gateway.stampede.tacc.utexas.edu<http://gateway.stampede.tacc.utexas.edu>

**Comment by mwvaughn** [comment]
Resolved in `master`. Will be bundled with next release.

**Comment by zyndagj** [comment]
https://github.com/cyverse/cyverse-sdk/blob/master/src/templates/tacc-stampede2-storage.jsonx

f5608d176f7c68e9eb18b4bccf37a065a1e02afb resolves #18
